### DISCUSSION TITLE
Possibility of releasing 1.29.1 having the container CI

### DISCUSSION THREAD

**Discussion by SayakMukhopadhyay** [mlflow/mlflow#6870] [Ideas]
Hello

I noticed that docker containerization and deployment has been implemented. I was looking into it a while back and had almost developed our own system parallely. Would it be possible to release a patch so that the GH Actions can be triggered thus creating the container images. It seems like the implementation was finished just a day after 1.29.0 was released which kinda sucks.

**Comment by JHibbard** [comment]
Thanks for the feedback, this is a good idea! Would this feature be something you'd be interested in helping to contribute?

**Reply by SayakMukhopadhyay** [reply]
I believe all the tooling are in place to do this. The only thing needed is to create a new release and the GH actions workflows should be triggered. See PRs #6731 and #6732

**Reply by JHibbard** [reply]
Hi SayakMukhopadhyay,

This is coming soon but won't be available for v1.29.X or v1.30.X, ideally by MLflow 2.0 we'll have the official images available through GitHub's container registry going forward.
